### ISSUE TITLE
Colour tools

### ISSUE THREAD

**Issue by gingerbeardman** [inseven/symbolic#265]
Colours are always difficult.

It would be cool to provide some tools to assist colour picking.

- simple pick of random colour based on hash of selected icon
- use a third-party index of colours
  - https://paletton.com
- or gradients
  - https://coolors.co/gradients
  - https://uigradients.com/
- etc

**Comment by gingerbeardman** [comment]
I'm trying this app

- https://apps.apple.com/gb/app/gradient-palette/id6446215235

It's "OK"

**Comment by jbmorley** [comment]
I'd been wanting to revisit all the color pickers, so I think these are fantastic suggestions. Thank you!

**Comment by jbmorley** [comment]
@gingerbeardman Do you have any references for the color theory behind things like Paletton—feels like it's following a bunch of very consistent and established rules that I could apply in my own picker.

**Comment by jbmorley** [comment]
I find it interesting that things like Gradient Palette just ship with a massive library of named gradients—do we think they're creating those themselves, or borrowing from known sets?

**Comment by gingerbeardman** [comment]
Not that I know of, but...

https://www.prasannamestha.com/blog/science-of-beautiful-color-gradients

**Comment by jbmorley** [comment]
OooOOOOooooooo 🧐

**Comment by gingerbeardman** [comment]
Another one

https://buttondown.com/allstationsbeyond/archive/space-cargo-tetris-palette-generation/

**Comment by gingerbeardman** [comment]
I guess this was enough for an app of its own.

https://bjango.com/mac/pinwheel/

**Comment by jbmorley** [comment]
Makes sense—seems like doing something other than ripping off a bunch of palettes from other people represents quiet a bit of work.
